exercises/practice/connect/src/main/java/Connect.java

```diff
@@ -0,0 +1,16 @@
+class Connect {
+
+    public Connect(String[] board) {
+        throw new UnsupportedOperationException("Implement this function");
+    }
+
+    public Winner computeWinner(String[] board) {
+        throw new UnsupportedOperationException("Implement this function");
+    }
+
+    enum Winner {
```

Comment:
I thought we were moving this enum to an isolated file and marking it as an editor only file (ie. the student should not be able to modify it in the UI).

Reply:
Oh thanks for the reminder